res/layout/whats_new_activity.xml

```diff
@@ -14,47 +14,48 @@
     </android.support.v4.view.ViewPager>
 
     <LinearLayout
-        android:orientation="horizontal"
         android:layout_width="match_parent"
         android:layout_height="0dp"
-        android:layout_weight="10"
-        android:weightSum="3"
-        android:layout_marginBottom="@dimen/standard_margin"
         android:layout_marginLeft="@dimen/standard_margin"
-        android:layout_marginRight="@dimen/standard_margin">
+        android:layout_marginRight="@dimen/standard_margin"
+        android:layout_marginBottom="@dimen/standard_margin"
+        android:layout_weight="11"
```

Comment:
The sum of this LinearLayout weight (11) and the one below with `android:layout_weight="90"` is more than what is defined in the parent view `android:layout_weight="100"` , any reason for that?